mne/viz/nirs.py

```diff
@@ -0,0 +1,162 @@
+"""Functions to visualise NIRS data."""
+
+import numpy as np
+
+from . import plot_alignment
+
+
+def plot_nirs_source_detector(data, info=None, radius=0.001,
```

Comment:
would you agree to kill this function and just add nirs support to plot_alignment?

Reply:
Definitely agree to killing this function.

But I would like to add a function to overlay channel analysis over a brain.

Would you prefer I make this PR for the extension to plot_alignment and move discussion of a data overlay to another issue/PR? Or move to MNE-NIRS?